vscodejs/index.js

```diff
@@ -20,10 +20,25 @@
  * ```
  * This pattern is captured by `E.withEditor`.
  */
+const path = require("path")
```

Comment:
I think all of these are great additions! 🚀

Please expose them at the top-level:

```
/** This accomplishes goals ABC. Example uses are as follows... */
E.getConfigDir = () => {...}

/** This accomplishes goals ABC. Example uses are as follows... */
E.isWindows = () => ...

/** This accomplishes goals ABC. Example uses are as follows... */
E.userConfigDir = () => ...

/** This accomplishes goals ABC. Example uses are as follows... */
E.userDirc = () => ...
```

Reply:
Got it.